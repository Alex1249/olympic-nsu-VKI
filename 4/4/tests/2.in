30
0 2
2 1 0
0 21
1 10
1 7
2 13 1
1 10
1 12
0 97
2 14 1
2 3 0
1 0
0 8
2 5 1
0 70
1 62
1 50
1 18
2 28 1
3 3 50
1 6
1 16
2 15 0
1 57
1 68
1 56
1 41
2 4 1
2 5 0
0 69
