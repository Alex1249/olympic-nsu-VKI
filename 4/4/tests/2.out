0
0
0
0
0
0
0
0
some
0
0
0
0
0
0
